### PR TITLE
Remove border on theme to avoid buttons in card to get a border. Use style for card instead

### PR DESCRIPTION
**Pull request by hugokallstrom**
Please see the button "Add payment method" in the info card below. This PR removes the border.
device-2021-10-05-151538
device-2021-10-05-151453

- [ ] Functionality is covered by an integration test
- [ ] Functionality is accessible in engineering mode

### CODE REVIEW

**Comment by hugokallstrom** [comment]
> I could use a little bit more context on this issue 😅

Sorry! Attached screenshots.

File: app/src/main/res/layout/home_info_card.xml [hunk 1]
```diff
@@ -65,7 +66,7 @@
 
         <Button
             android:id="@+id/action"
-            style="?materialButtonTextStyle"
```
Comment (hedvigoscar) [inline app/src/main/res/layout/home_info_card.xml:68]: Is this correct? 🤔

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -118,4 +118,10 @@
         <item name="android:paddingTop">12dp</item>
     </style>
 
+    <style name="Hedvig.Theme.InfoCard">
```
Comment (hedvigoscar) [inline app/src/main/res/values/styles.xml:121]: This style does not contain the background color for warning cards, and it appears to be used for warning cards, but is named info card 🤔

**Review by hedvigoscar — COMMENTED** [review]
A few questions came to mind!

File: app/src/main/res/layout/home_info_card.xml (hunk 1 above)
Comment (hugokallstrom) [reply app/src/main/res/layout/home_info_card.xml:68]: Oh we have this `<item name="materialButtonTextStyle">@style/Hedvig.Button.Text</item>` - changing back.

File: app/src/main/res/values/styles.xml (hunk 2 above)
Comment (hugokallstrom) [reply app/src/main/res/values/styles.xml:121]: So this style is only for setting the card properties. For setting the colors we are still using the themes, like this:

```
android:theme="@style/ThemeOverlay.Hedvig.RenewalCard"
style="@style/Hedvig.Theme.InfoCard"
```

Comment (hugokallstrom) [reply app/src/main/res/values/styles.xml:121]: Having a theme at the top level with `<item name="strokeWidth">1dp</item>` will apply a stroke on every child, in this case the buttons.

Comment (hedvigoscar) [reply app/src/main/res/values/styles.xml:121]: Sensible.

**Review by hedvigoscar — APPROVED** [review]
Approved with previously mentioned nits

**Pull request merged**